apps/omg_eth/test/omg_eth/eth_test.exs

```diff
@@ -23,90 +23,37 @@ defmodule OMG.EthTest do
   """
 
   alias OMG.Eth
-  alias OMG.Eth.Encoding
 
   use ExUnitFixtures
   use ExUnit.Case, async: false
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
-  @eth OMG.Eth.RootChain.eth_pseudo_address()
-
-  @moduletag :wrappers
   @moduletag :common
 
-  @tag fixtures: [:eth_node]
-  test "get_ethereum_height and get_block_timestamp_by_number return integers" do
-    assert {:ok, number} = Eth.get_ethereum_height()
-    assert {:ok, timestamp} = Eth.get_block_timestamp_by_number(number)
-    assert is_integer(number)
-    assert is_integer(timestamp)
-  end
-
-  @tag fixtures: [:contract]
-  test "get contract deployment height", %{contract: contract} do
-    {:ok, number} = Eth.RootChain.get_root_deployment_height(contract.txhash_contract, contract.contract_addr)
-    assert is_integer(number)
+  setup do
+    {:ok, _} = Application.ensure_all_started(:ethereumex)
+    vcr_path = Path.join(__DIR__, "../fixtures/vcr_cassettes")
+    ExVCR.Config.cassette_library_dir(vcr_path)
+
+    # NOTE achiurizo
+    #
+    # this is a hack to ensure we reset the counter to 0 despite
+    # the fixtures now resetting the counter.
+    :ets.insert(:rpc_requests_counter, {:rpc_counter, 0})
```

Comment:
is this still needed?

Reply:
Yes we should leave this in the event we need to record additional cassettes.  I can re-word the description.